scripts/check-package-resolved-policy.py

```
@@ -38,6 +39,20 @@
+
+class PackageNode(NamedTuple):
+    """Dependency edges declared by one Package.swift."""
+
+    has_url_dependency: bool
+    path_dependencies: list[str]
+    # Normalized text of every `.package(url:)` call, so a version-requirement
+    # bump on an unchanged URL still counts as a remote-resolution change.
+    url_calls: frozenset[str]
+
+
+EMPTY_PACKAGE_NODE = PackageNode(False, [], frozenset())
```

Ruff (0.16.0)

[warning] Boolean positional value in function call

(FBT003)